go.mod

```diff
@@ -20,13 +21,12 @@ require (
 	go.opentelemetry.io/otel/exporters/stdout/stdouttrace v1.16.0
 	go.opentelemetry.io/otel/sdk v1.27.0
 	go.opentelemetry.io/otel/trace v1.27.0
-	go.temporal.io/api v1.36.0
-	go.temporal.io/sdk v1.28.1
+	go.temporal.io/api v1.38.0
+	go.temporal.io/sdk v1.29.1
 	go.temporal.io/sdk/contrib/datadog v0.2.0
 	go.temporal.io/sdk/contrib/opentelemetry v0.6.0
 	go.temporal.io/sdk/contrib/opentracing v0.1.0
 	go.temporal.io/sdk/contrib/tally v0.2.0
-	go.temporal.io/sdk/contrib/tools/workflowcheck v0.2.0
```

Comment:
I'm not sure why this was removed, I ran `go get -u github.com/kisielk/errcheck` at some point and it got removed.